### ISSUE TITLE
You can appear to block yourself, if you try the error message is not found

### ISSUE THREAD

**Issue by shanaqui** [HabitRPG/habitica-android#1449]
You get the error message "string 'blockYourself' not found", including if you do not have the app translated. I see this on my version too.

Device: Sony F5321
Android Version: 25
AppVersion: Version 3.1 (2676)
BETA
User ID: 33bb14bd-814d-40cb-98a4-7b76a752761c
Level: 81
Class: rogue
Is in Inn: false
Uses Costume: true
Custom Day Start: 16
Timezone Offset: -60

**Comment by mike-the-monk** [comment]
I'd like to claim this one.

I looked up the strings, and the string blockYourself doesn't actually exist, even in the English. Should be a fairly simple fix.

**Comment by saraolson** [comment]
@Mike-Antonacci all yours! marked it in progress for you

**Comment by Alys** [comment]
@Mike-Antonacci I'm sorry, this issue isn't as simple as it appears. It needed a tech admin to work that out so there wasn't a way for shanaqui or saraolson to realise this.

If the staff post to say anything that disagrees with what I put here, listen to them not me.

The error message for trying to block yourself comes from the API, rather than being configured in the Android app, so it needs to be fixed in the API, which means a pull request for the main repo (https://github.com/HabitRPG/habitica/). I'm about to move this issue to the main repo. I think there then should be automatic redirection when you try to access this issue.

I'll keep it as in progress for you in the main repo because this should be a simple fix there of just adding the string, and a running local install would not be needed in this case (although I usually wouldn't recommending doing a PR there without being able to test!)  However tell us if you'd rather not work on it and we'll set it to help wanted.

The file to edit in the main repo would be `groups.json` in the `website/common/locales/en` directory, and I'd recommend adding the new string near the `unblock` string to keep similar texts together. As for the Android app, you would not need to add the same string to the other language files.

How I worked this out was by doing `git grep blockYourself` in the Android app's repo. No results were returned, which means (at least in this case) that the string is not used in any of the Android app's code. (NB another reason for no results can be that a string is generated programmatically from multiple smaller strings.)

I then did `git grep blockYourself` in the main repo and saw the results below. This means that the string IS used in that repo, and the fact that there's no result for a locales file shows the problem.

```
/code/habitica $ git grep blockYourself
test/common/ops/blockUser.test.js:32:      expect(error.message).to.eql(i18n.t('blockYourself'));
website/common/script/ops/blockUser.js:9:  if (req.params.uuid === user._id) throw new BadRequest(i18n.t('blockYourself', req.language));


# for Android, no grep result:
/code/habitica $ cd ../habitica-android/
/code/habitica-android $ git grep blockYourself
/code/habitica-android $ 
```

It's very interesting that the `test/common/ops/blockUser.test.js` test file (which you can see in the grep results above) doesn't throw an error. That's something we'll want to look into but it's not directly related to fixing this issue. If you decide you don't want to work on this, I'll add the test file bug to this issue for a coder to fix both; if you do go ahead with this, I'll make a new issue for the test bug.
